lib/remote/apilistener.cpp

```diff
@@ -1651,12 +1659,8 @@ Value ApiListener::HelloAPIHandler(const MessageOrigin::Ptr& origin, const Dicti
 						case 1:
 							break;
 						default:
-							Log(LogWarning, "ApiListener")
-								<< "Unexpected Icinga version of endpoint '" << endpoint->GetName() << "': "
-								<< nodeVersion / 100u / 100u << "." << nodeVersion / 100u % 100u << ".x"
-								<< " . Expected one of: " << l_AppVersionInt / 100u / 100u << "."
-								<< l_AppVersionInt / 100u % 100u << ".x" << ", " << (l_AppVersionInt / 100u - 1u) / 100u
-								<< "." << (l_AppVersionInt / 100u - 1u) % 100u << ".x";
+							PrintAppVersion(nodeVersion,Log(LogWarning, "ApiListener")
```

Comment:
Make `Log(LogWarning, "ApiListener")` a variable `Log log (LogWarning, "ApiListener");` and continue with `log << ...`.